### PR TITLE
unify cob3-X config and launch

### PR DESCRIPTION
**Pull request by fmessmer**
related to #311

### CODE REVIEW

File: cob_bringup/robots/cob3-9.xml [hunk 1]
```diff
@@ -49,42 +52,54 @@
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="name" value="top" />
 		</include>
-
-		<include file="$(find cob_bringup)/drivers/battery_voltage_filter.launch" >
+		<include file="$(find cob_bringup)/drivers/relayboard.launch" >
```
Comment (NadiaHG) [inline cob_bringup/robots/cob3-9.xml]: the relayboard is attached to pc2

File: cob_bringup/robots/cob3-9.xml [hunk 2]
```diff
@@ -49,42 +52,54 @@
...
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		
+		<include file="$(find cob_bringup)/drivers/base_driver.launch" >
```
Comment (NadiaHG) [inline cob_bringup/robots/cob3-9.xml]: the base is attached to pc2

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-9.launch [hunk 3]
```diff
@@ -134,26 +134,30 @@
...
 		<param name="topic_name" value="/pc3_usage"/>
 	</node>
-	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-11)" cwd="node" respawn="false" output="screen" >
+	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-10)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="Wifi Status (ddwrt)"/>
 		<param name="topic_name" value="/wifi_status"/>
 	</node>
+	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-11)" cwd="node" respawn="false" output="screen" >
+		<param name="diagnostics_name" value="base_controller"/>
+		<param name="topic_name" value="/base/joint_states"/>
```
Comment (NadiaHG) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-9.launch:159]: are you sure about the topic name?

File: cob_hardware_config/cob3-2/config/teleop.yaml [hunk 4]
```diff
@@ -9,6 +9,7 @@ arm_joint12_button: 0
 arm_joint34_button: 1
 arm_joint56_button: 2
 arm_joint7_button: 3
+base_safety_button: 4
```
Comment (NadiaHG) [inline cob_hardware_config/cob3-2/config/teleop.yaml:12]: are you sure that the cob3-2 joystick version has this button and in this position?

File: cob_hardware_config/cob3-2/config/teleop.yaml [hunk 5]
```diff
@@ -30,25 +31,19 @@ modules: {
     joint_step: [0.1]
   },
   torso: {
-  joint_names: ['torso_lower_neck_pan_joint','torso_lower_neck_tilt_joint','torso_upper_neck_pan_joint','torso_upper_neck_tilt_joint'],
-  joint_step: [0.05,0.05,0.075,0.075]
-#    joint_names: ['torso_lower_neck_tilt_joint','torso_pan_joint','torso_upper_neck_tilt_joint'],
-#    joint_step: [0.05,0.075,0.075]
+    joint_names: ['torso_lower_neck_pan_joint','torso_lower_neck_tilt_joint','torso_upper_neck_pan_joint','torso_upper_neck_tilt_joint'],
```
Comment (NadiaHG) [inline cob_hardware_config/cob3-2/config/teleop.yaml:34]: supports already teleop actuators movements?

**Comment by fmessmer** [comment]
all modifications have been made offline/conceptual in order to unify the three robots and in order to group reladet nodes for a clearer structure...
No guarantee it works!
If some parts need to run on specific PCs, I wasn't aware of that and the respective changes should be reverted (or preferably harmonized differently!)

File: cob_bringup/robots/cob3-9.xml (hunk 1 above)
Comment (fmessmer) [inline cob_bringup/robots/cob3-9.xml]: ok then, did not know the hardware setup

File: cob_bringup/robots/cob3-9.xml (hunk 2 above)
Comment (fmessmer) [inline cob_bringup/robots/cob3-9.xml]: ok then, did not know the hardware setup

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-9.launch (hunk 3 above)
Comment (fmessmer) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-9.launch:159]: yes, in simulation all platforms use the new plugins with the new namespaces and topic names

File: cob_hardware_config/cob3-2/config/teleop.yaml (hunk 4 above)
Comment (fmessmer) [inline cob_hardware_config/cob3-2/config/teleop.yaml:12]: no, needs to be checked when cob_teleop is revised...

File: cob_hardware_config/cob3-2/config/teleop.yaml (hunk 5 above)
Comment (fmessmer) [inline cob_hardware_config/cob3-2/config/teleop.yaml:34]: no, we need to specify,when cob_teleop is revised...

**Comment by NadiaHG** [comment]
Visual Review: OK

**Pull request merged**